frontend/source/javascript/components/vpc/list.js

```diff
@@ -0,0 +1,104 @@
+'use strict';
+
+angular
+    .module('cloud-inquisitor.components')
+    .component('vpcList', {
+        bindings: {
+            params: '<',
+            result: '<'
+        },
+        controller: vpcListController,
+        controllerAs: 'vm',
+        templateUrl: 'vpc/list.html'
+    })
+;
+
+vpcListController.$inject = ['MetadataService', 'Utils'];
+function vpcListController(MetadataService, Utils) {
+    const vm = this;
+    vm.vpcs = undefined;
+    vm.vpcCount = 0;
+    vm.accounts = MetadataService.accounts;
+    vm.regions = MetadataService.regions;
+    vm.filters = {};
+    vm.updateFilters = updateFilters;
+    vm.resetFilters = resetFilters;
+    vm.updatePath = updatePath;
+    vm.$onInit = onInit;
+
+    //region Functions
+    function onInit() {
+        vm.result.$promise.then(onLoadSuccess, Utils.onLoadFailure);
+    }
+
+    function onLoadSuccess(response) {
+        vm.vpcs = response.vpcs;
+        vm.vpcCount = response.vpcCount;
+        vm.filters = {
+            accounts: {
+                type: 'select',
+                label: 'Accounts',
+                multiple: true,
+                values: vm.accounts.map(v => v.accountName),
+                selected: vm.params.accounts
+            },
+            regions: {
+                type: 'select',
+                label: 'Regions',
+                multiple: true,
+                values: vm.regions,
+                selected: vm.params.regions
+            },
+            vpcId: {
+                type: 'select',
+                label: 'VPC ID',
+                multiple: true,
+                values: vm.vpcs.map(v => v.properties.vpcId),
+                searchable: true,
+                selected: vm.params.vpcId
+            },
+            cidrV4: {
+                type: 'select',
+                label: 'IPv4 Prefix',
+                multiple: true,
+                values: vm.vpcs.map(v => v.properties.cidrV4),
+                searchable: true,
+                selected: vm.params.cidrV4
+            },
+            vpcFlowLogsStatus: {
+                type: 'select',
+                label: 'VPC Flow Log Status',
+                multiple: false,
+                values: {
+                    'active' : 'ACTIVE',
+                    'inactive' : '',
```

Comment:
This empty label could cause some issues in the dropdown

Reply:
Yeah agreed when we update we set to undefined so this will cause selecting any to stick unless you reset and reload the page. I'm removing this...the AWS API doesn't document any values other than ACTIVE or null, however I do recall seeing an unhealthy state but can not find the response so this should be fine for now.

Reply:
I'm also going to modify the collector to mark any null FlowLog value as NOT CONFIGURED and we'll handle that way, since and unhealthy state also implies issue with configuration.